src/app/page.js

```diff
@@ -1,3 +1,13 @@
+'use client';
```

Comment:
🔴  again, using 'use client' disables ssr on top level for that page, is that ok?

Reply:
![image](https://github.com/keenethics/mayak/assets/57231356/9c93769c-7440-4653-9101-02dc72f4adf5)

look at the source. Global styles and styles are already arriving.

Reply:
I checked if SSR works like this. If I'm wrong, let me know.

Reply:
You can avoid using "use client" here by moving MyDiv styled component into "_components" folder. Then you'll be able to just import "MyDiv" whenever you want, including server side "Home" component. Give it a try.

Reply:
page hydration happens on the server side, and returns HTML with styles to the client.

Reply:
@watchlar2000 I know it. This is just a small example for demonstration.